rpcs3/rpcs3qt/tooltips.h

```diff
@@ -52,6 +52,7 @@ class Tooltips : public QObject
 		const QString vulkan_async_scheduler       = tr("Determines how to schedule GPU async compute jobs when using asynchronous streaming.\nUse 'Safe' mode for more spec compliant behavior at the cost of some CPU overhead. This setting works with all devices.\nUse 'Fast' to use a faster but hacky version. This option is internally disabled for NVIDIA GPUs due to causing GPU hangs.");
 		const QString disable_msl_fast_math        = tr("Disables Fast Math for MSL shaders, which may violate the IEEE 754 standard.\nDisabling it may fix some artifacts, especially on Apple GPUs, at the cost of performance.");
 		const QString suspend_savestates           = tr("When this mode is on, emulation exits when saving and the savestate file is concealed after loading it, preventing reuse by RPCS3.\nThis mode is like hibernation of emulation: if you don't want to be able to cheat using savestates when playing the game, consider using this mode.\nDo note that the savestate file is not gone completely, just ignored by RPCS3. You can manually relaunch it if needed.");
+		const QString compatible_savestates        = tr("When this mode is on, SPU Emulation is optimized for savestates suitability and compatibility instead at the cost of slight performance downgrade.\nIf this mode is off, attempting to make a savestate may result in an SPU pause error in the log that would disallow doing so for some games.");
```

Comment:
When this mode is on, SPU emulation prioritizes savestate compatibility, however, it may reduce performance slightly. \nWhen this mode is off, some games may not allow making a savestate and show an SPU pause error in the log.